judge/views/ticket.py

```diff
@@ -160,6 +163,10 @@ def post(self, request, *args, **kwargs):
         ticket = self.get_object()
         if ticket.is_open != self.open:
             ticket.is_open = self.open
+            logger.warn("New ticket for {problem}: {url}".format({
```

Comment:
This will log every time the ticket is closed/opened. You probably want it in `NewProblemTicketView.form_valid`. Also, use `logger.info` instead of `logger.warn`.